qa/ccs-common-rest/src/test/java/org/elasticsearch/test/rest/yaml/CcsCommonYamlTestSuiteIT.java

```diff
@@ -184,6 +194,11 @@ private static ClientYamlTestCandidate rewrite(ClientYamlTestCandidate clientYam
                     modifiedExpectedValue = rewriteExpectedIndexValue(matchSection.getExpectedValue());
                 }
                 rewrittenSection = new MatchAssertion(matchSection.getLocation(), matchSection.getField(), modifiedExpectedValue);
+            } else if (section instanceof IsFalseAssertion falseAssertion) {
+                if (lastAPIDoSection.startsWith("async_") && ((IsFalseAssertion) section).getField().endsWith("_clusters")) {
+                    // in ccs scenarios, the response "_cluster" section will be there
+                    rewrittenSection = new IsTrueAssertion(falseAssertion.getLocation(), falseAssertion.getField());
```

Comment:
interesting! don't we need the same for search? I guess we never verify that the section is not there when not using ccs :)

Reply:
No, I think we don't. These rewrite rules will need some tweaking over time, but covering all possible scenarios ahead of time they might get used is most likely out of scope I think.